docs/docsite/rst/userguide/rbac.rst

```diff
@@ -0,0 +1,347 @@
+
+
+.. _rbac-legacy-ug:
+
+Legacy RBAC model
+------------------
+
+.. index::
+   single: roles   
+   pair: legacy; RBAC
+
+RBACs are easiest to think of in terms of Roles which define precisely who or what can see, change, or delete an "object" for which a specific capability is being set. RBAC is the practice of granting roles to users or teams.
+
+There are a few main concepts that you should become familiar with regarding AWX's RBAC design--roles, resources, and users. Users can be members of a role, which gives them certain access to any resources associated with that role, or any resources associated with "descendant" roles.
+
+A role is essentially a collection of capabilities. Users are granted access to these capabilities and AWX's resources through the roles to which they are assigned or through roles inherited through the role hierarchy.
+
+Roles associate a group of capabilities with a group of users. All capabilities are derived from membership within a role. Users receive capabilities only through the roles to which they are assigned or through roles they inherit through the role hierarchy. All members of a role have all capabilities granted to that role. Within an organization, roles are relatively stable, while users and capabilities are both numerous and may change rapidly. Users can have many roles.
+
+
+Role Hierarchy and Access Inheritance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Imagine that you have an organization named "SomeCompany" and want to allow two people, "Josie" and "Carter", access to manage all the settings associated with that organization. You should make both people members of the organization's ``admin_role``.
+
+|user-role-relationship|
+
+.. |user-role-relationship| image:: ../common/images/user-role-relationship.png
+
+Often, you will have many Roles in a system and you will want some roles to include all of the capabilities of other roles. For example, you may want a System Administrator to have access to everything that an Organization Administrator has access to, who has everything that a Project Administrator has access to, and so on. 
+
+This concept is referred to as the 'Role Hierarchy':
+
+- Parent roles get all capabilities bestowed on any child roles
+- Members of roles automatically get all capabilities for the role they are a member of, as well as any child roles.
+
+The Role Hierarchy is represented by allowing Roles to have "Parent Roles". Any capability that a Role has is implicitly granted to any parent roles (or parents of those parents, and so on).
+
+|rbac-role-hierarchy|
+
+.. |rbac-role-hierarchy| image:: ../common/images/rbac-role-hierarchy.png
```

Comment:
The DAB RBAC system actually explicitly ditches the "role hierarchy" system. It's actually really specific about this. The old system used the role hierarchy for everything, but the new system is different.

In DAB RABC, to spell it out, the "Project Admin" role doesn't need to exist for other organization-level roles to give permissions to a project. This is accomplished by the role permission list instead. So the organization admin role has project CRUD permissions (like change_project, delete_project) in its permission list. There is still somewhat of a hierarchy, but at the resource level, in that organizations are the parent object (RBAC-wise) of projects.

Reply:
The information about the new RBAC only exists in section 9.1 only. Section 9.2 pertains to _**only**_ legacy RBAC stuff. I left them both in for those who still need reference to the old RBAC.